granian/cli.py

```diff
@@ -204,16 +204,16 @@ def option(*param_decls: str, cls: Optional[Type[click.Option]] = None, **attrs:
 @option(
     '--reload-ignore-dirs',
     help=(
-        'Names of directories to ignore changes for (i.e. should not trigger reload). '
-        'Extends the default list of directories to ignore in watchfiles.filters.DefaultFilter.'
+        'Names of directories to ignore changes for. '
+        "Extends the default list of directories to ignore in watchfiles' default filter"
     ),
     multiple=True,
 )
 @option(
     '--reload-ignore-patterns',
     help=(
-        'Entity patterns (regex) to ignore changes for. '
-        'Extends the default list of patterns to ignore in watchfiles.filters.DefaultFilter.'
+        'Path patterns (regex) to ignore changes for. '
```

Comment:
Quick note because I stumbled over this while testing the feature.

I had also assumed that this will regex match on the full path of something that triggered the reload. However, this only matches the last part of the path, meaning the exact file/dir that triggered the reload, not the entire path.

https://watchfiles.helpmanual.io/api/filters/#watchfiles.DefaultFilter.ignore_entity_patterns

"Entity" isn't a much better description 😛 but "Path" might be suggesting a different behaviour altogether. WDYT?

<sub>I want to avoid issues where users report this to be broken.

Reply:
Oh, I missed that. Would *Relative path patterns* make sense to you?

Reply:
Sorry for the late answer.

I think being fully explicit could help.
```suggestion
        'File/directory name patterns (regex) to ignore changes for. '
```
WDYT?

Reply:
Looks perfect. Can you open a PR for this? otherwise I'll do it by myself later

Reply:
No worries, will do!